smac/smbo/smbo.py

```diff
@@ -122,9 +122,19 @@ def run(self):
             # get all found configurations sorted according to acq
             challengers = self.choose_next(X, Y)
 
-            time_spend = time.time() - start_time
-            logging.debug(
-                "Time spend to choose next configurations: %.2f sec" % (time_spend))
+            time_spent = time.time() - start_time
```

Comment:
I think the functionality should be moved into it's own method. This will keep the SMBO loop simple and easy to read. Also, it will allow for a simple unit test of the functionality in which you don't have to mess around with the logger.